submissions/Zakkarat/html-forms/index.html

```diff
@@ -0,0 +1,460 @@
+                <th class="th-to-left">FILE NAME</th>
+                <th class="th-to-left">FILE SIZE</th>
+                <th class="th-to-left">NUMBER OF WORDS</th>
+                <th class="th-to-left">REMOVE</th>
+              </tr>
+              <tr>
+                <td class="image-doc">buisness_operations.docx</td>
+                <td>78 kB</td>
+                <td>3894 words</td>
+                <td class="td-to-center">
+                  <img
+                    tabindex="0"
+                    src="./img/waste-bin.svg"
+                    class="deleteFile"
+                  />
+                </td>
+              </tr>
+              <tr>
+                <td>buisness_financial.docx</td>
+                <td>23 kB</td>
+                <td>2123 words</td>
+                <td class="td-to-center">
+                  <img
+                    tabindex="0"
+                    src="./img/waste-bin.svg"
+                    class="deleteFile"
+                  />
+                </td>
+              </tr>
+            </table>
+          </section>
+          <section class="documentLanguage">
+            <div class="allTextDocumentUpload">
+              <span class="numberSection"><b>2</b></span>
+              <h2 class="uploadText">Choose your languages pairings</h2>
+            </div>
+            <div class="block-to-center">
+              <div class="select-button select-button-from">
+                <select name="language" class="select-button-inner" required>
+                  <option value="english" selected>English</option>
+                  <option value="Arabian">Arabian</option>
+                  <option value="Kudrish">Kudrish</option>
+                  <option value="Ukrainian">Ukrainian</option>
+                </select>
+              </div>
+            </div>
+            <div class="block-to-center">
+              <div class="select-button select-button-to">
+                <select name="language" class="select-button-inner" required>
+                  <option value="english">English</option>
+                  <option value="Arabian" selected>Arabian</option>
+                  <option value="Kudrish">Kudrish</option>
+                  <option value="Ukrainian">Ukrainian</option>
+                </select>
+              </div>
+              <div class="select-button select-button-next">
+                <select name="language" class="select-button-inner" required>
+                  <option value="english">English</option>
+                  <option value="Arabian">Arabian</option>
+                  <option value="Kudrish">Kudrish</option>
+                  <option value="Ukrainian" selected>Ukrainian</option>
+                </select>
+              </div>
+              <img
+                tabindex="0"
+                src="./img/waste-bin.svg"
+                class="deleteFile select-delete"
+              />
+              <button type="button" class="add-button">
```

Comment:
Make your buttons more user friendly(`hover`,`active`)

Reply:
Done!

Reply:
@Zakkarat i cant see changes on your demo(delete buttons don't have any visual effects) also you can improve UI of  this  `+971 (0) 555 555
help@qtest.com`  links